Prediction modelling update

FlyingCuteCats opened the pull request:
1. A logistic model is created with 3 basic predictors.

2. Different combinations of predictors are tested and conclusion made.

FlyingCuteCats committed:
Prediction modelling update

1. A logistic model is created with 3 basic predictors.

2. Different combinations of predictors are tested and conclusion made.

Arg-Pep commented:
- Do we need to export this file as csv? Can we not do this at the end only after all the data wrangling and analysis? 

![image](https://github.com/Arg-Pep/DS406---Group-Project/assets/151948016/da57708a-7095-49c5-b1af-8470786e9d59)

- The analysis and conclusions should be in the report file I created on Teams (we discussed this in our last call). **This file is only to show the code.** The report is a separate document and should be 4 pages long including graphs and tables. Please see the introduction I did, add your text accordingly and ensure there is some flow.

- These are too many outputs which make the message confusing. It's better to have one output per model, probably where we have AIB and BIC, and comment on these. One output per fitted model should be enough. Keep it simple.

![image](https://github.com/Arg-Pep/DS406---Group-Project/assets/151948016/c658f785-b674-4ece-be2c-e00de2c6d6cf)


- There seem to be multiple models done. Some with one predictor, some with several. The report is only 4 pages long. We need to reduce it to 2 or a maximum of 3 models and make clear which one is the best and why.

- I can't see clearly in the file what is our final prediction or output. What is the product most likely to be resolved with monetary relief? 

- I think the code is good in general, but just needs to be cleaned and reduced to the essentials only. Please see what I have already explained in the report introduction and just add what's needed in the methodology and analysis of the report to answer the research questions.

markal33 committed:
Create Group_project_writeup.docx

Initial write up for Data modelling & Analysis

FlyingCuteCats committed:
Analysis updated

I deleted the ANOVA part because we agreed to use the simple AIC to compare models. I added a block (see  Block 107) to summarise which combination of product, issue and state has most monetary relief cases. Block 109 is a new dataframe showing the TPR and TNP in training and test sets under different thresholds. Block 110 is a ROC curve graph but I'm not sure how to read it. Also updated some Methodology and Analysis sessions in the report.
